core/Http.php

```diff
@@ -1096,6 +1096,14 @@ private static function getProxyConfiguration($url)
         $proxyPort = Config::getInstance()->proxy['port'];
         $proxyUser = Config::getInstance()->proxy['username'];
         $proxyPassword = Config::getInstance()->proxy['password'];
+        $proxyExclude = Config::getInstance()->proxy['exclude'];
+
+        if (!empty($proxyExclude)) {
+            $excludes = explode(",", $proxyExclude);
```

Comment:
Might be good to apply some simple filtering to avoid some common configuration problems (like additional spaces, unneeded commas, ...). We are doing that for most other comma separated values as well:
```suggestion
            $excludes = explode(',', $proxyExclude);
            $excludes = array_map('trim', $excludes);
            $excludes = array_filter($excludes);
```